snuba/manual_jobs/job_loader.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any
+
+import click
+
+from snuba.manual_jobs import Job
+
+
+class JobLoader:
+    @staticmethod
+    def get_job_instance(class_name: str, dry_run: bool, **kwargs: Any) -> "Job":
+        job_type_class = Job.class_from_name(class_name)
+        if job_type_class is None:
+            raise click.ClickException(
+                f"Job does not exist. Did you make a file {class_name}.py yet?"
+            )
```

Comment:
nit: this should be replaced with a non-Click exception (since this method may not be called from the CLI `snuba jobs` in the future). But we've gone through a number of rounds on this and I think it can wait for a later iteration